assets/brunch-config.js

```diff
@@ -0,0 +1,70 @@
+const neat = require('bourbon-neat').includePaths;
```

Comment:
Possible undefined variable: 'require' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Reply:
Ok in Node